src/appleseed/renderer/kernel/lighting/bdpt/bdptlightingengine.h

```diff
@@ -48,7 +49,9 @@ class BDPTLightingEngineFactory
 {
   public:
     // Constructor.
-    explicit BDPTLightingEngineFactory(const ParamArray& params);
+    explicit BDPTLightingEngineFactory(
```

Comment:
You can safely remove `explicit` now that the constructor takes more than one argument. I just wish C++ had explicit constructors by default :)

Reply:
Ah, you are right. Will do it.